clients/src/main/java/org/apache/kafka/clients/admin/Admin.java

```diff
@@ -930,6 +931,120 @@ ElectLeadersResult electLeaders(
         Set<TopicPartition> partitions,
         ElectLeadersOptions options);
 
+
+    /**
+     * Change the reassignments for one or more partitions.
+     * Providing an empty Optional (e.g via {@link Optional#empty()}) will <bold>cancel</bold> the reassignment for the associated partition.
```

Comment:
maybe we could say it will "reset" the reassignment?

Reply:
hmm, cancel seems like a better description?

Reply:
Right, `reset` isn't a good word - I was thinking of `revert` when I typed it. I don't have a strong opinion on this